src/components/migrated-bch.js

```diff
@@ -0,0 +1,324 @@
+        // try again after unlocking
+        needsUnlock = true;
+      } else {
+        throw e;
+      }
+    }
+
+    if (needsUnlock) {
+      try {
+        await bitgo.unlock({ otp: this.state.twofa });
+        recoveryTx = await migratedWallet.submitTransaction({
+          txHex: recoveryTx.hex
+        });
+      } catch (e) {
+        // failed even after unlock - this is fatal
+        console.log('got error on submit after unlock');
+        console.error(e);
+        throw e;
+      }
+    }
+    this.setState({ recoveryTx, recovering: false });
+  };
+
+  saveTransaction = () => {
+    const fileData = this.state.recoveryTx;
+    let fileName;
+    const filePrefix = this.props.bitgo.env === 'prod' ? 'bch' : 'tbch';
+
+    fileName = `${filePrefix}r-${moment().format('YYYYMMDD')}.signed.json`;
+
+    const dialogParams = {
+      filters: [{
+        name: 'Custom File Type',
+        extensions: ['json']
+      }],
+      defaultPath: '~/' + fileName
+    };
+
+    // Retrieve the desired file path and file name
+    const filePath = dialog.showSaveDialog(dialogParams);
+    if (!filePath) {
+      // TODO: The user exited the file creation process. What do we do?
+      return;
+    }
+
+    try {
+      fs.writeFileSync(filePath, JSON.stringify(fileData, null, 4), 'utf8');
+    } catch (err) {
+      console.log('error saving', err);
+      this.setState({ error: 'There was a problem saving your recovery file. Please try again.' });
+    }
+  };
+
+  render() {
+    const coin = this.props.bitgo.env === 'prod' ? 'bch' : 'tbch';
+
+    return (
+      <div>
+        <h1 className='content-header'>Migrated Bitcoin Cash Recoveries</h1>
+        <p className='subtitle'>This tool will help you recover Bitcoin Cash from migrated wallets which are no longer officially supported by BitGo.</p>
+        <Alert color='warning'>
+          Do we need a warning here?
+        </Alert>
+        <hr />
+        <Form>
+          <InputField
+            label='Original Bitcoin Wallet ID'
+            name='walletId'
+            onChange={this.updateRecoveryInfo}
+            value={this.state.walletId}
+            tooltipText={formTooltips.walletId}
+            disallowWhiteSpace={true}
+          />
+          <InputField
+            label='Destination Address'
+            name='recoveryAddress'
+            onChange={this.updateRecoveryInfo}
+            value={this.state.recoveryAddress}
+            tooltipText={formTooltips.recoveryAddress}
+            disallowWhiteSpace={true}
+            format='address'
+            coin={this.props.bitgo.coin(coin)}
+          />
+          <InputField
+            label='Wallet Passphrase'
+            name='passphrase'
+            onChange={this.updateRecoveryInfo}
+            value={this.state.passphrase}
+            tooltipText={formTooltips.passphrase}
+            isPassword={true}
+          />
+          <InputField
+            label='2FA Code'
```

Comment:
If we decide to save transactions for support instead of submitting them, we can get rid of this field.

Reply:
I'm thinking this route might be the way to go. Less error prone, and more control on our end. Some of these recoveries could involve significant amounts of value (that is, if BCH ever recovers 🤣)

Reply:
LOL. Do these recoveries bypass policies? If so, they should absolutely go through support as they will want to do extra verification for transactions higher than the policy on the wallet would normally allow. Even if they don't, it's probably a good thing for support to manually approve large recovery transactions for customers who haven't taken the time to set up policies.

Reply:
No, they do not bypass policies. That would be bad. I still think going through support is a better idea though. Will update the flow for that use case.